DSCResources/MSFT_xDNSServerAddress/MSFT_xDNSServerAddress.psm1

```diff
@@ -48,15 +48,14 @@ function Get-TargetResource
         $($LocalizedData.GettingDNSServerAddressesMessage)
         ) -join '')
 
+    $dnsClientServerAddresses = Get-NetAdapter -Name $InterfaceAlias | ForEach-Object { Get-DnsClientServerAddress -InterfaceAlias $_  -AddressFamily $AddressFamily }
```

Comment:
[Style guidance](https://github.com/PowerShell/DscResources/blob/master/StyleGuidelines.md) of our resources. So we're just adhearing to it, it's to make code reviews easier on smaller monitors I think.